api/proto/teleport/legacy/client/proto/event.proto

```diff
@@ -206,5 +206,7 @@ message Event {
     // IdentityCenterAccountlAssignment is a resource representing a potential
     // Permission Set grant on a specific AWS account.
     teleport.identitycenter.v1.AccountAssignment IdentityCenterAccountAssignment = 74;
+    // GitServer is a resource for Git proxy server.
+    types.ServerV2 GitServer = 75;
```

Comment:
Why do we need a dedicated event? This will not play well when mixing auth server versions.

Reply:
as commented in other PR, i cheated a little by reusing the same `types.ServerV2`. But it's still considered as a new resource. 

Do you suggest using existing `types.ServerV2 Server = 11;`? wouldn't that cause more confusion for older servers?

Reply:
The overloading of ServerV2 is a different problem, but I don't believe the watchers would misbehave here; the filtering is done by asking for the correct `WatchKind.Kind`, much like you can watch for node, proxy and/or auth and get events for the three different kinds independently.

Reply:
yes, you are right. i see your point now. reverted